var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -25,14 +25,13 @@ class Neuron(Package):
 
     # Patch which reverts 81a7a39 for numerical compat
     patch('revert_Import3d_numerical_format.patch', when='@7.8.0:')
-    # Patch which reverts d9605cb for not hanging on ExperimentalMechComplex
-    patch('revert_d9605cb.patch', when='@7.8.0a')
 
     # Patch which applies load balancing fix
-    patch('apply_79a4d2af_load_balance_fix.patch', when='@7.8.0:')
+    patch('apply_79a4d2af_load_balance_fix.patch', when='@7.8.0b')
 
     version('develop', branch='master')
-    version('7.8.0b',  commit='92a208b', preferred=True)
+    version('7.8.1',   commit='79a4d2a', preferred=True)
```

Comment:
Latest is always preferred.
```suggestion
    version('7.8.1',   commit='79a4d2a')
```

Reply:
Good point but in this case `preferred` should be defined explicitly otherwise version `2018-10` is selected